packages/flutter/lib/src/rendering/sliver_fill.dart

```diff
@@ -82,14 +118,14 @@ class RenderSliverFillViewport extends RenderSliverFixedExtentBoxAdaptor {
     double leadingScrollOffset,
     double trailingScrollOffset,
   }) {
-    final double padding = _padding;
-    return childManager.estimateMaxScrollOffset(
+    // Only overriding the documentation.
+    return super.estimateMaxScrollOffset(
       constraints,
       firstIndex: firstIndex,
       lastIndex: lastIndex,
-      leadingScrollOffset: leadingScrollOffset - padding,
-      trailingScrollOffset: trailingScrollOffset - padding,
-    ) + padding + padding;
+      leadingScrollOffset: leadingScrollOffset,
+      trailingScrollOffset: trailingScrollOffset,
```

Comment:
Why can we just ignore the padding?

Reply:
The padding is now accounted for in `widget.estimateMaxScrollOffset` (in `packages/flutter/lib/src/widgets/sliver.dart`)